Simplenote/AccountVerificationRemote.swift

```diff
@@ -2,34 +2,16 @@ import Foundation
 
 // MARK: - AccountVerificationRemote
 //
-class AccountVerificationRemote {
-    private let urlSession: URLSession
-
-    init(urlSession: URLSession = URLSession.shared) {
-        self.urlSession = urlSession
-    }
-
+class AccountVerificationRemote: Remote {
     /// Send verification request for specified email address
     ///
-    func verify(email: String, completion: @escaping (_ success: Bool) -> Void) {
+    func verify(email: String, completion: @escaping (_ result: Result) -> Void) {
```

Comment:
Can we switch to [Swift's Result Type](https://developer.apple.com/documentation/swift/result)?

Reply:
Totally.  I have gone ahead and setup to use the Swift Result instead of the custom one built into remote.

I haven't used that before, so check it out and let me know if there are any changes that need to be made. @jleandroperez